### ISSUE TITLE
[1.12.2] Infinite Falling Blocks with Cubic Chunk installed

### ISSUE THREAD

**Issue by DonatiFilippo** [Teneted/Tenet#1886]
**Minecraft Version :** 1.12.2

**Mohist Version :** The issues starts after build 247 (245 doesn't present this) and continues up to the latest version
**Operating System :** I tested this on several OS, mainly Ubuntu and Windows

**Concerned mod / plugin** :  Cubic Chunks https://www.curseforge.com/minecraft/mc-mods/opencubicchunks/files
**Logs :** No log needed

**Steps to Reproduce :**
 1. Start a server with a Cubic Chunks world in it (note: install the mod first)
 2. Place a block affected by gravity (ex: sand, gravel, anvil etc...) in a position where the will normally fall
 3. Watch it keeping falling and being teleported back to the intial point in loop

**Description of issue :** I first noticed this bug when you relised the first compatible version with Cubic Chunks but fortunatly, I managed to find a work around  by setting both config in the entity.yml file to true. The problem is that after updating to the 247 build (I keep mohist always updated so I can confirm that in 245 the bug isn't present) even this work around stoped working (also as a good mesure after noticing the bug I resetted the entity.yml config with no resault)

**Comment by MohistBOT** [comment]
**Hello, thanks for opening this issue!**

Before continuing, please understand that Mohist is a **free** and **open source** software, made by people in **their free time**. Know that we are not able to give any eta about when your issue will get answered or fixed.

> Our team will first check if your issue isn't duplicated. If it's the case, we will close your issue, giving your the number of the other issue so that you can follow its progress. If you are able to do it, please check that your issue isn't duplicated, this would help us a lot.

*Thank you for your patience, and have a nice day (or night)!*

**Comment by DonatiFilippo** [comment]
By looking at the commit history, I noticed how https://github.com/MohistMC/Mohist/commit/59ecb1331e994bf38bd3fffe42fe555dfea8a8db#diff-e84dab6b4aad61a12c6ab50a5054341e7a2c399a4475b2e657f2be563a8a5ed8L662 ditched the entity,yml config. This explain why the work around stop working but not why the error is happening in the first place

**Comment by DonatiFilippo** [comment]
in addition I saw that this issues is the same reported here (https://github.com/MohistMC/Mohist/issues/1650) just with a bunch of more info, like Cubic Chunks being the root cause of the problem. Also to clarify, differently from what Agrrox and Coppertine said the error happens even near 0,0

**Comment by Coppertine** [comment]
Bumping this as this has been an issue since the first version of Mohist being able to use CubicChunks,
As well, due to the log4j exploit, people are resorting of using later versions of Mohist which disables entity.yml and still includes said issue which has been fixed by mojang on later versions. However, since CubicChunks is only for 1.12.2 as of now, players are forced with glitching falling sand blocks.

**Comment by Joziah2** [comment]
I would like to add that I have experienced this same issue, as well as some weird desync issue with dropped items.

Happens even when CubicChunks is the only mod on the server. Falling sand won't despawn/land, and dropped items behave very strangely, not leaving the inventory when dropped, seeming to come out of the wrong position vertically, and sometimes spawning clones that can't be picked up.

My experiences were with Mohist 299, although I also tried 245 and 235 with the same result.

I hope this issue can be fixed! Thanks a lot!
